cloud-sql.mk

```diff
@@ -7,24 +7,27 @@ install-cloud-sql-proxy:
 		curl -o cloud_sql_proxy https://dl.google.com/cloudsql/cloud_sql_proxy.darwin.amd64 && \
 		chmod +x cloud_sql_proxy
 
-test-cloud-sql-config:
+batch-secrets/batch-test-cloud-sql-config:
 	mkdir -p batch-secrets && kubectl get secret \
 	    batch-test-cloud-sql-config -n batch-pods \
 	    -o "jsonpath={.data.batch-test-cloud-sql-config\.json}" \
 	  | base64 --decode \
 	  | jq '.host = "127.0.0.1" | .port = $(CLOUD_SQL_PORT)' \
 	    > batch-secrets/batch-test-cloud-sql-config.json
 
-production-cloud-sql-config:
+batch-secrets/batch-production-cloud-sql-config:
```

Comment:
this should be batch-secrets/batch-production-cloud-sql-config.json same above

Reply:
I don't really like having this long make target name. Is this standard for Makefiles?

Reply:
Yeah, it is quite long; I don't know of a better way, except maybe making it a variable, say

```
PROD_CONF := batch-secrets/batch-production-cloud-sql-config.json

$(PROD_CONF):
```

variables work perfectly well as targets. Maybe @danking would have another suggestion?